### PR TITLE
docs: i18n(fr): Updating French introduction

### PR DESCRIPTION
**Pull request by thomasbnt**
## Description
Updating French introduction. Linked to this issue https://github.com/open-sauced/intro/issues/145.

### Todo: Files
- [x] README
- [x] Introduction
- [x] Qu'est ce que l'Open Source
- [x] Outils pour réussir
- [x] Comment contribuer à l'Open Source
- [x] La sauce secrète
- [x] Conclusion
- [x] Ressources additionnelles
- [x] Glossaire

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update → French translation
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
Close https://github.com/open-sauced/intro/issues/145

## Tier (staff will fill in)

- [ ] Tier 1
- [ ] Tier 2
- [ ] Tier 3
- [ ] Tier 4

### CODE REVIEW

**Comment by nickytonline** [comment]
I can help review this if you need @BekahHW.

File: translations/fr/01-introduction.md [hunk 1]
```diff
@@ -1,73 +1,61 @@
...
 
-Tout au long de ce cours, nous fournirons des exemples et des démonstrations pour aider à illustrer les concepts clés et fournir des opportunités pratiques de pratique.
-Dans la mesure du possible, nous présenterons des projets et des outils open source réels, tels que [OpenSauced](https://opensauced.pizza/), pour montrer comment appliquer les compétences que vous avez acquises.
+Tout au long de ce cours, nous fournirons des exemples et des démonstrations pour aider à illustrer les concepts clés et fournir des opportunités pratiques de pratique. Dans la mesure du possible, nous présenterons des projets et des outils open source réels, tels que [OpenSauced](https://opensauced.pizza/), pour montrer comment appliquer les compétences que vous avez acquises.
 
 Veuillez noter que bien que ces exemples et démos soient conçus pour vous guider et vous inspirer, votre propre parcours open source peut différer en fonction de vos intérêts, de vos compétences et des projets choisis.
 
 Utilisez les exemples et les démonstrations comme point de départ, mais n'ayez pas peur d'explorer d'autres projets, outils et ressources à mesure que vous continuez à apprendre et à grandir.
 
 ## LFG 🚀
```
Comment (adiati98) [inline translations/fr/01-introduction.md:53]: We've updated this to "Let's Get Started 🚀"

**Review by adiati98 — COMMENTED** [review]
Hey @thomasbnt ,
Thanks so much for working on this! 🙌🏼

I've left a review here. Also, can you check the `_sidebar.md` and update the titles to reflect the English version? Thank you! 😊 

@nickytonline, thanks for the offer! Really appreciate it. 💙

If you have the time, it would be great if you could help with the review! TIA! 😄

**Comment by adiati98** [comment]
OMG. Sorry, @thomasbnt !
I wasn't paid attention. Thought I was quoting your message instead 😂

File: translations/fr/01-introduction.md (hunk 1 above)
Comment (thomasbnt) [reply translations/fr/01-introduction.md]: I forgot to translate this one, good catch!

**Comment by adiati98** [comment]
Hey @thomasbnt ,
Heads up! We've just merged PR #153 and #154.
Would you please update the French translation accordingly?
Thank you in advance! 😊

**Comment by nickytonline** [comment]
Apologies, but I don't have the bandwidth at the moment to review this. I've got some feature work I need to tend to before we head to our offsite. Sorry about that.

**Comment by thomasbnt** [comment]
> Apologies, but I don't have the bandwidth at the moment to review this. I've got some feature work I need to tend to before we head to our offsite. Sorry about that.

No problem! Have a good weekend Nick!

**Review by adiati98 — APPROVED** [review]
Hey @thomasbnt,
Thanks for updating our French translation!
We'll merge this in 🚀

**Review by BekahHW — APPROVED** [review]

**Comment by thomasbnt** [comment]
🙌🥳

**Pull request merged**